tests/backup/backup_partial_test.go

```diff
@@ -471,3 +474,323 @@ var _ = Describe("{PartialBackupSuccessWithPxVolumes}", func() {
+		})
+
+		Step("Create scheduled backups for backup validation", func() {
+			log.InfoD("Creating %v scheduled backups for namespaces [%s]", scheduledBackupCount, appNamespaces)
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			log.InfoD("Creating a new schedule policy for new apps for backup validation")
+			schedulePolicyName = fmt.Sprintf("backup-schedule-%v", RandomString(6))
+			schedulePolicyInfo := Inst().Backup.CreateIntervalSchedulePolicy(5, int64(15), 5)
+			err = Inst().Backup.BackupSchedulePolicy(schedulePolicyName, uuid.New(), BackupOrgID, schedulePolicyInfo)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of new schedule policy [%s]", schedulePolicyName))
+			schedulePolicyUid, err = Inst().Backup.GetSchedulePolicyUid(BackupOrgID, ctx, schedulePolicyName)
+			log.FailOnError(err, "Fetching uid of schedule policy [%s]", schedulePolicyName)
+			var wg sync.WaitGroup
+			for i := 1; i <= scheduledBackupCount; i++ {
+				scheduledBackupName := fmt.Sprintf("scheduled-backup-%v-%v", i, RandomString(6))
+				log.Infof("Creating scheduled backup [%s]", scheduledBackupName)
+				wg.Add(1)
+				go func(i int) {
+					defer wg.Done()
+					_, err = CreateScheduleBackupWithoutCheckWithVscMapping(scheduledBackupName, SourceClusterName, backupLocationName, backupLocationUid, appNamespaces, labelSelectors, BackupOrgID, "", "", "", "", schedulePolicyName, schedulePolicyUid, ctx, map[string]string{}, true)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup with schedule [%s]", schedulePolicyName))
+					scheduledBackups = append(scheduledBackups, scheduledBackupName)
+				}(i)
+				time.Sleep(timeBetweenScheduledBackups)
+			}
+			wg.Wait()
+		})
+
+		Step("Verify if the first scheduled backup is Success", func() {
+			log.InfoD("Verifying if the first scheduled backup is of Success status")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			var wg sync.WaitGroup
+			for _, schBkpName := range scheduledBackups {
+				wg.Add(1)
+				go func(schBkpName string) {
+					defer wg.Done()
+					firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, schBkpName, BackupOrgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching name of the first scheduled backup with schedule [%s]", schBkpName))
+					log.Infof("Validating if the first scheduled backup [%s] of the schedule [%s] is a success", firstScheduleBackupName, schBkpName)
+					err = BackupSuccessCheck(firstScheduleBackupName, BackupOrgID, MaxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second, ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying if the backup [%s] of schedule [%s] is a Success", firstScheduleBackupName, schBkpName))
+				}(schBkpName)
+			}
+			wg.Wait()
+		})
+
+		Step("Start a watcher to delete the data export CR of the first namespace to create a partial backup", func() {
```

Comment:
I have created a function for this `WatchAndKillDataExportCR`, once you rebase on master you will get it

Reply:
I saw that, I will accommodate the changes. Thank you.